Zajecia/src/main/java/pl/isa/javasmugglers/App.java

```diff
@@ -1,13 +1,12 @@
 package pl.isa.javasmugglers;
 
-/**
- * Hello world!
- *
- */
+
 public class App 
 {
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
+    public static void main( String[] args ){
+        //testowe odpalenie aplikacji
+        MenuGlowne menuGlowne = new MenuGlowne();
```

Comment:
Trzymajcie się standardu nazewnictwa klas - a konkretnie wersji językowej.
Jak masz klasy np. User, DatabaseWriter itd, to wszystkie powinny być w tej samej konwencji.